### PR TITLE
Docs fix errors common

### PR DESCRIPTION
**Pull request by petebacondarwin**
This fixes all the "dgeni" errors for the `@angular/common` API docs.

### CODE REVIEW

File: packages/common/src/pipes/number_pipe.ts [hunk 1]
```diff
@@ -63,11 +63,9 @@ export class DecimalPipe implements PipeTransform {
  * Uses the function {@link formatPercent} to format a number as a percentage according
  * to locale rules.
  *
- * ### Example
+ * **Example**
```
Comment (gkalpak) [inline packages/common/src/pipes/number_pipe.ts]: This shows directly below `Description`. Change to `Usage:` or something?

File: packages/common/src/pipes/number_pipe.ts [hunk 2]
```diff
@@ -101,11 +99,9 @@ export class PercentPipe implements PipeTransform {
  * Uses the functions {@link getCurrencySymbol} and {@link formatCurrency} to format a
  * number as currency using locale rules.
  *
- * ### Example
+ * **Example**
```
Comment (gkalpak) [inline packages/common/src/pipes/number_pipe.ts]: This shows directly below `Description`. Change to `Usage:` or something?

**Review by gkalpak — APPROVED** [review]

File: packages/common/src/pipes/number_pipe.ts (hunk 1 above)
Comment (petebacondarwin) [reply packages/common/src/pipes/number_pipe.ts]: Actually I think that a nice sentence looks better.

**Comment by petebacondarwin** [comment]
@IgorMinar I have updates this PR to fix docs issues in the `common` package as we discussed. PTAL.

**Review by IgorMinar — APPROVED** [review]
Looks good. thanks!

**Pull request closed without merging**